docs/src/1_Mechanics/06_Bestiary.yaml

```diff
@@ -72,6 +72,19 @@ Wolf:
     - Attack, Weapon
     - Pack Tactics
 
+Summoned Creature:
+  Type: Companion
+  Level: 1
+  HP: 3
+  AR: 3
+  Attribs:
+    AGL: 1
+    STR: 1
+    INT: -1
+    CON: -1
+  Powers:
+    - Attack, Weapon
```

Comment:
Does this have range? Or melee only?

Reply:
As written its melee only, especially as the first level of it. But I could see a future skill tree having several different variants of 'Summoned Creature' that grant access to additional powers and ranged abilities.

Reply:
RAW, I think this is range 6, given that range is a property of the power that is 6 when unspecified. Does range need to instead be a property of the item? Does there need to be two base versions of `Attack, Weapon`? 

If we make a `Attack, Melee Weapon` and `Attack, Ranged Weapon`, we might discuss the unique features of each beyond range, and how DofA balances risk/reward of range as a whole

Reply:
I think as a high level resolution the Attack, Weapon is only purchased once as a Power but the Weapon Property depends on the range. So a Sword can only be used in Range 1, Shortbow is a range 6, Longbow Range 8, etc. 

The broader discussion of melee VS ranged is a good one for us to consider as part of the playtesting and gauge the difference players make based on what's available to them in their inventory. 

I can add an issue integrating Items YAML with properties tied to a Power that we can tackle on a future PR.